champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/mage/passives/Immolate.java

```diff
@@ -199,10 +205,10 @@ private void sendState(Player player, boolean state) {
     public void loadSkillConfig() {
         baseFireTickDuration = getConfig("baseFireTickDuration", 4.0, Double.class);
         fireTickDurationIncreasePerLevel = getConfig("fireTickDurationIncreasePerLevel", 0.0, Double.class);
-
         baseFireTrailDuration = getConfig("baseFireTrailDuration", 2.0, Double.class);
         fireTrailDurationIncreasePerLevel = getConfig("fireTrailDurationIncreasePerLevel", 0.0, Double.class);
-
-        speedStrength = getConfig("speedStrength", 1, Integer.class);
+        speedStrength = getConfig("speedStrength", 0, Integer.class);
+        strengthLevel = getConfig("speedLevel", 1, Integer.class);
```

Comment:
...

Reply:
It technically works; better than some of my issues

Reply:
NICE ONE STUDENBT!